src/pages/en/core-concepts/astro-components.md

```diff
@@ -173,6 +181,14 @@ const items = ["Dog", "Cat", "Platypus"];
 </ul>
 ```
 
+Fragments can also be useful to avoid wrapper elements when adding [`client:*` directives](/en/reference/directives-reference/), as in the following example:
```

Comment:
Looks good now, but hadn’t noticed this before. I don’t think you can ever add a _client_ directive to a fragment because they’re not hydrated. I think only `set:html` or `set:text` make sense with fragments so we can probably do this:

```suggestion
Fragments can also be useful to avoid wrapper elements when adding [`set:*` directives](/en/reference/directives-reference/#sethtml), as in the following example:
```